plugins/modules/sensor_policy_info.py

```diff
@@ -0,0 +1,289 @@
+            type: str
+            returned: success
+            sample: "Windows 10 Sensor Policy"
+          assignment_rule:
+            description: The assignment rule of the group.
+            type: str
+            returned: success
+            sample: "hostname:['demo-win10-1']"
+          created_by:
+            description: The user who created the group.
+            type: str
+            returned: success
+            sample: "user@example.com"
+          created_timestamp:
+            description: The timestamp when the group was created.
+            type: str
+            returned: success
+            sample: "2021-03-01T00:00:00Z"
+          modified_by:
+            description: The user who last modified the group.
+            type: str
+            returned: success
+            sample: "user@example.com"
+          modified_timestamp:
+            description: The timestamp when the group was last modified.
+            type: str
+            returned: success
+            sample: "2021-03-01T00:00:00Z"
+      created_by:
+        description: The user who created the policy.
+        type: str
+        returned: success
+        sample: "user@example.com"
+      created_timestamp:
+        description: The timestamp when the policy was created.
+        type: str
+        returned: success
+        sample: "2021-03-01T00:00:00Z"
+      modified_by:
+        description: The user who last modified the policy.
+        type: str
+        returned: success
+        sample: "user"
+      modified_timestamp:
+        description: The timestamp when the policy was last modified.
+        type: str
+        returned: success
+        sample: "2021-03-01T00:00:00Z"
+      settings:
+        description: The settings of the policy.
+        type: dict
+        returned: success
+        sample: {
+          "build": "n-1|tagged"
+        }
+headers:
+    description:
+      - The HTTP headers returned from the API.
+    type: dict
+    returned: always
+    sample: {
+        "X-Ratelimit-Limit": "6000",
+        "X-Ratelimit-Remaining": "5999"
+    }
+meta:
+    description:
+      - The metadata returned from the API.
+      - Contains pagination information.
+    type: dict
+    returned: always
+    sample: {
+        "pagination": {
+            "limit": 5000,
+            "offset": 0,
+            "total": 1
+        },
+        "query_time": 0.012
+    }
+'''
+
+import copy
+import traceback
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.basic import missing_required_lib
+from ansible_collections.crowdstrike.falcon.plugins.module_utils.args_common import AUTH_ARG_SPEC
+from ansible_collections.crowdstrike.falcon.plugins.module_utils.falconpy_utils import get_falconpy_credentials
+
+
+try:
+    from falconpy import SensorUpdatePolicy
+except ImportError:
+    HAS_FALCONPY = False
+    FALCONPY_IMPORT_ERROR = traceback.format_exc()
+else:
+    HAS_FALCONPY = True
```

Comment:
Required for sanity checks to pass. We should do something similar to `kubernetes.core` and abstract this out of the modules. This will also allow us to check for minimum and maximum versions etc.